streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -191,7 +192,7 @@ void suspendTasksAndState()  {
 
     private RuntimeException unAssignChangeLogPartitions() {
         try {
-            // un-assign the change log partitions
+            // TODO verify: this call is save an cannot throw; code can be simplified
```

Comment:
safe?

Reply:
I think that is right, we can get rid of the catch block.